site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -425,14 +466,21 @@
         ">   $\\frac{\\text{true positives}}{\\text{true positives + false positives}}$\n",
         "*   **Recall** is the percentage of **actual** positives that were correctly classified\n",
         ">   $\\frac{\\text{true positives}}{\\text{true positives + false negatives}}$\n",
+        "\n",
+        "**Note:** Accuracy is not a helpful metric for this task. You can have 99.8%+ accuracy on this task by predicting False all the time.  \n",
+        "\n",
+        "#### Other metrices\n",
+        "\n",
+        "The following metrics take into account all possible choices of thresholds $t$, but they are not proper scoring rules and only assess the ranking of predictions, not their absolute values.\n",
```

Comment:
This sentence is hard to understand without a little more context on "proper scoring rules".

Reply:
Do you have a suggestion?
I added one sentence for proper scoring rules above. Then this clearly says that AUC only assesses the ranking of predictions. Otherwise stated: Best AUC does not guarantee to be close to the true probabilities.

Reply:
Good call introducing proper scoring earlier.

> only assess the ranking of predictions, not their absolute values.

I don't understand this. Which ranking & values are we talking about?

> Best AUC does not guarantee to be close to the true probabilities.

Right, but if all you want is a deterministic classifier, we don't care about the true probabilities.